### PR TITLE
First version of element library  template (#430)

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #430.

### CODE REVIEW

File: doc/extension/Pipfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+verify_ssl = true
+name = "pypi"
+
+[packages]
+taipy-gui = ">=2.0"
+
+[dev-packages]
+
+[requires]
+python_version = "3"
```
Comment (FredLL-Avaiga) [inline doc/extension/Pipfile:12]: `>= 3.8` ?

File: doc/extension/README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+- If you are using `pipenv`:
+   ```sh
+   $ pipenv --python $PYTHON_VERSION
+   $ pipenv shell
+   $ pipenv install
+   ```
+
+- If you are not using `pipenv`:
+   ```sh
+   $ python -m venv ./venv
```
Comment (FredLL-Avaiga) [inline doc/extension/README.md:76]: Where does venv come from?

**Review by FredLL-Avaiga — APPROVED** [review]
Don't forget to rename js to webui

File: doc/extension/README.md (hunk 2 above)
Comment (FabienLelaquais) [reply doc/extension/README.md:76]: Looks like it comes out of the box with Python... who would have guessed?

File: doc/extension/Pipfile (hunk 1 above)
Comment (FabienLelaquais) [reply doc/extension/Pipfile:12]: I guess that makes a lot of sense. Hopefully this syntax is allowed...

Comment (FabienLelaquais) [reply doc/extension/Pipfile:12]: It is not. Only a specific version (3.8, 3.9...) is.
Therefore... 3 works here

**Review by dinhlongviolin1 — APPROVED** [review]
Nicely Done!

**Review by FredLL-Avaiga — APPROVED** [review]
go

**Pull request merged**